### PR TITLE
Update time limit

### PR DESCRIPTION
**Pull request by LucyMHall**
- a user should now only be able to update a post for 10 mins after they created it, if the current time is after this then they are shown an error message and given a link to follow back to their posts page

### CODE REVIEW

File: app/views/posts/edit.html.erb [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
-  <%= form.label :message %>
-  <%= form.text_area :message, size: "15x3", :value => @post.message %>
-  <%= form.submit "Submit" %>
+
+
+<% @ten_mins_in_seconds = 600 %>
+<% @post_creation_time = @post.created_at%>
+<% created_at_time_plus_10mins = @post_creation_time + @ten_mins_in_seconds %>
+
+<% if Time.now > created_at_time_plus_10mins %>
```
Comment (Danielandro) [inline app/views/posts/edit.html.erb]: I like this conditional. Never used flash messages before

File: config/application.rb [hunk 2]
```diff
@@ -14,5 +14,7 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.time_zone = "London"
```
Comment (Danielandro) [inline config/application.rb]: This is dope.

File: spec/features/user_can_only_update_their_post_within_10min_of_creation_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+RSpec.feature "Update my posts", type: :feature do
+
+  scenario "User tries to update post more than 10 mins after creation" do
+    sign_up
+    click_link "View Your Posts"
+    create_a_post("Hello")
+
+    post_creation_date = Post.find(1).created_at
+    eleven_mins_in_seconds = 660
+    allow(Time).to receive(:now) {post_creation_date + eleven_mins_in_seconds}
```
Comment (Danielandro) [inline spec/features/user_can_only_update_their_post_within_10min_of_creation_spec.rb]: Stubbing the method, this is cool

**Comment by lucianmot** [comment]
you dropped some science alright!

**Pull request merged**